### PR TITLE
FAQ: My application was not accepted, what now?

### PR DESCRIPTION
**Pull request by ocefpaf**
I had this on my notes since last year. I removed a part about funds that does not apply this year but maybe we could make it generic and add it anyway. It is something like this:

```
- For the in site events the organizers try to optimize for the funds available.
We may invite some non-funded students in case you have your own funds.

Maybe we should add an entry in the application about this, something like:
* I'm have my own funds to attend the OHW or I'm applying to the funds available.
```

### CODE REVIEW

File: _pages/01-applicants.md [hunk 1]
```diff
@@ -112,4 +112,14 @@ please mention that in the application.
 
 Yes. While we understand that not all topics may be 100% relevant to you we want to maximize engagement. Note that we post the lessons and videos online later, so if you are interested in only a portion of the OceanHackWeek consider that an option rather than taking someone's spot.
 
----
+**Q: I did not get accept, what now?**
```
Comment (emiliom) [inline _pages/01-applicants.md:115]: Change "accept" to "accepted"

Comment (ocefpaf) [reply _pages/01-applicants.md:115]: I guess I did not push the latest changes. The question should be the same as the issue title.

File: _pages/01-applicants.md [hunk 2]
```diff
@@ -112,4 +112,14 @@ please mention that in the application.
...
+
+Sadly resources are limited and we cannot accommodate all the applicants we would like.
+
+It is important to note that a rejected application does not mean a bad one! The reasons are:
+
+- The organizer tries to accommodate a good balance between novices, intermediaries and experts. That means a candidate may be rejected one year and accepted in another due to the different pool of applicants. Rule of thumb? Always apply again if possible! Also, if you believe you can contribute to the OHW as a helper instead of a student, please get in touch! We are always in need of helpers.
```
Comment (emiliom) [inline _pages/01-applicants.md:121]: Change "intermediaries" to "intermediates" (not quite right, but better than intermediaries!)

File: _pages/01-applicants.md [hunk 3]
```diff
@@ -112,4 +112,14 @@ please mention that in the application.
...
+
+- One important factor that goes into the selection of candidates is the motivation paragraph in your application. Boilerplate answers or short answers like: "I want to participate" or "It is important to me" may get lower ranking than those who read the past year materials and wrote a paragraph on how the OHW can really make a difference to them. TL;DR be honest, be you!
```
Comment (emiliom) [inline _pages/01-applicants.md]: Here's a potential suggested changed that I'm not 100% sold on: "In short: be informed, be honest, be you!". It sounds more intimidating than what you have there, but it's also more helpful and does reflect what's said in the longer text. What do you think?

I've also changed the "TL;DR" bit. I don't think that lingo is universally known.

Comment (ocefpaf) [reply _pages/01-applicants.md]: I don't think that it is intimidating. Let's add that.

**Comment by emiliom** [comment]
Thanks @ocefpaf!

> I had this on my notes since last year. 

I recognize that text! I'm glad you had it on hand.

> I removed a part about funds that does not apply this year but maybe we could make it generic and add it anyway. It is something like this:
>
> - For the in site events the organizers try to optimize for the funds available. We may invite some non-funded students in case you have your own funds.

I think that statement sounds confusing, and I'm not sure what the message should be. I would vote for adding it now if we were all on board with the wording. So my vote is for adding it later (which of course just means we'll forget ...).

I think we should also add a statement about programming expectations. 

- Also, please refer to the FAQ question: *Q: I am proficient in Matlab / R / other languages but have not used Python. It has been on my list to learn. How can I make my application be stronger?*

**Comment by ocefpaf** [comment]
> I think that statement sounds confusing, and I'm not sure what the message should be. I would vote for adding it now if we were all on board with the wording. So my vote is for adding it later (which of course just means we'll forget ...).

We can not at it at all if we tweak the 2021 application to have a field of self-support vs applying for support. Let's leave this problem for future us.

**Comment by emiliom** [comment]
It looks great. I can merge, or go ahead and self-merge. Thanks!

**Pull request merged**